single_asset/tezos_fa2_single_tests/test_simple.py

```diff
@@ -1,77 +0,0 @@
-from pathlib import Path
```

Comment:
What happened to these tests?

Reply:
removed. They were really redundant, checking more pytezos testing harness rather than actual FA2